### DISCUSSION TITLE
Windows 10 mkcert error NET::ERR_CERT_COMMON_NAME_INVALID microsoft edge

### DISCUSSION THREAD

**Discussion by themanfrommsu** [FiloSottile/mkcert#435] [Q&A] [answered]
downloaded mkcert using chocolatey into windows 10 machine.

Installed mkcert into windows 10:
mkcert -install

generated cert:
mkcert -key-file xxxx.local.key -cert-file xxxx.local.crt xxxx.local

added xxxx.local into the hosts file on the windows 10 machine.

Now when try to open connection to "https://xxxx.local" with any web client, gives warning that connection isn't private.  With Firefox and Chrome it displays warning but still displays the page.  With Microsoft Edge, will not display the page and displays error message:

NET::ERR_CERT_COMMON_NAME_INVALID
Subject: mkcert development certificate
Issuer: mkcert CAMPUSAD\memememe@hostname (mylastname, myfirstname)
Expires on: Mar 21, 2024
Current date: Feb 10, 2022
PEM encoded chain:
-----BEGIN CERTIFICATE-----
MIIEXzCCAsegAwIBAgIRAOEbxPOh6mKRkYULxQtgZ58wDQYJKoZIhvcNAQELBQAw
gZcxHjAcBgNVBAoTFW1rY2VydCBkZXZlbG9wbWVudCBDQTE2MDQGA1UECwwtQ0FN
UFVTQURcc2ltbW9uc3JASVRTLTcxU0c4WTIgKFNpbW1vbnMsIFJ5YW4pMT0wOwYD
VQQDDDRta2NlcnQgQ0FNUFVTQURcc2ltbW9uc3JASVRTLTcxU0c4WTIgKFNpbW1v
bnMsIFJ5YW4pMB4XDTIxMTIyMTIyNDIyNVoXDTI0MDMyMTIxNDIyNVowXDEnMCUG
A1UEChMebWtjZXJ0IGRldmVsb3BtZW50IGNlcnRpZmljYXRlMTEwLwYDVQQLDChD
QU1QVVNBRFxzaW1tb25zckBJVFMtNzFTRzhZMiAoc2ltbW9uc3IpMIIBIjANBgkq
hkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA2Kw5hyEpuRSXbjtWdxHmyFDM4VSN4S0E
wV0WvaGdNTZSzUBNiOpmMBDrAMqqly1Bpyje/dBvq3oiKSF5hoMH5nFXBZKBUj8C
Noch5+ntsDylkm5ohqLlfrO4coepfx5ItXQqxQZH0YASFj94h1ZuWdo7VSYIvJ6T
OF6Xn+YyDE3RfOMKTq5xWr+6PlWNJOFc7XKpelF4EuoacsBIsiG9ngwXmB51nHBw
o2mAju/+sz8EfPkK3HQxJEapCxcc4msofmBMkOMYBcRQj5i/cPttxZiMxjVK2JsS
xVIvoJYkdVkTKPWJ0Vdg+SjMWmTyY0nw1a3vsFtzHdLg5w4y5Lm3rQIDAQABo2Aw
XjAOBgNVHQ8BAf8EBAMCBaAwEwYDVR0lBAwwCgYIKwYBBQUHAwEwHwYDVR0jBBgw
FoAUXgWtIdZGTA8TKKV/QHYtooBmoKEwFgYDVR0RBA8wDYILbXVzZXVtLnRlc3Qw
DQYJKoZIhvcNAQELBQADggGBAAINhB207LoIHME5sq+vcMBu2nRHDtUIDOJQtuqz
2qb6qzUqD1mfebQP+j3kQl44rdr+iY0B2WkNrnBlC7XEbii6GPqu6A4LUVUMCaXm
+51rF7KuIa8nSHZfuweIceHEnSjAa+x9X65xvrEiRwABgjvICzxpmsePnSUlG4Xh
8dPrfVCQI06Lyice7bKMrYQkDAkeqQ9AlE2u/rXZT/wtfI33dDfAsH3f/OtLAJ8p
NSmIzxtcFrRxbbPbup9AThs/yS3fuMTj8pKd7z2eAsOEoc4p0/JzYekBYJvXwEJo
fsPJY2arqK5byl1vfdBRrQ8ETRInnq4BwwcdgKW6KX8jB/9eW9eUNPNkPRyLcdSz
3yXdtjO1GDRNeqA7NhI1kyrG14pgKquifUoQOFIGCGitaVRrgImkqkCtk+Dug/vp
cps8urrwd8QjbwQamh6I71vIpeH7VIYIYdypTKJldR2j4Jhy7z6toBoqNJMagBd4
5C3Qj1uaKMQaR7J3s5ZrZw+k9g==
-----END CERTIFICATE-----
-----BEGIN CERTIFICATE-----
MIIE/zCCA2egAwIBAgIQRqg16nI/I0UzuwfsK4AJuzANBgkqhkiG9w0BAQsFADCB
lzEeMBwGA1UEChMVbWtjZXJ0IGRldmVsb3BtZW50IENBMTYwNAYDVQQLDC1DQU1Q
VVNBRFxzaW1tb25zckBJVFMtNzFTRzhZMiAoU2ltbW9ucywgUnlhbikxPTA7BgNV
BAMMNG1rY2VydCBDQU1QVVNBRFxzaW1tb25zckBJVFMtNzFTRzhZMiAoU2ltbW9u
cywgUnlhbikwHhcNMjExMTIzMjAyMzI2WhcNMzExMTIzMjAyMzI2WjCBlzEeMBwG
A1UEChMVbWtjZXJ0IGRldmVsb3BtZW50IENBMTYwNAYDVQQLDC1DQU1QVVNBRFxz
aW1tb25zckBJVFMtNzFTRzhZMiAoU2ltbW9ucywgUnlhbikxPTA7BgNVBAMMNG1r
Y2VydCBDQU1QVVNBRFxzaW1tb25zckBJVFMtNzFTRzhZMiAoU2ltbW9ucywgUnlh
bikwggGiMA0GCSqGSIb3DQEBAQUAA4IBjwAwggGKAoIBgQC+0QaNoCE3jKwHhW9q
TWe1gLnoTM5nTi2pCzH+kTUn9qV2+rd3IDbGy3674ZeLhmoTjTEiQp8yYAmi82xa
vATA5x22WU9+4eQXfI0izXztlZxTKCoJhjrmguLWZ4JiyDzH/9r+/hNMdSkQ4M1j
aA6DlczbDstXpkbl1dNjZ4BjO21t+5NiP15mDV95UUsqYLeDtgz2zK2s0AUoWeMo
8hc5x22DEsxyWJ6uLc18GQF+H3nYVDHPYzI6u9Gpt05HAxBqmiuNBktALA/3b3PX
1V/M6mAA3Dmw3I/3vIQXG15P1Yrs+IGVPdGvlj7LGwp6xzqTz9D/NvGjUzdjzwFG
gyqy4KUJhWg5l6Abtl4bCLADeKIbOS8oD0hIcyXkEaeCX+fEit8ClPtkwkBEdAx+
Q+Scoths/EBdNfILljNRzBx07sz8LVhUeAh9P+5oEkJ1R9L03rsT606hMGjGnpRL
A1L5mbBiatosr1Axd2GPYuIacC/dYpCsLnFT29TSgdA+8RUCAwEAAaNFMEMwDgYD
VR0PAQH/BAQDAgIEMBIGA1UdEwEB/wQIMAYBAf8CAQAwHQYDVR0OBBYEFF4FrSHW
RkwPEyilf0B2LaKAZqChMA0GCSqGSIb3DQEBCwUAA4IBgQBYyFkI26HtutefGVh5
bdogLKEnOWP2N6wzQasflM9lg32IInl6wefLhPvIX/FDYAMaUT3/2ynw2nAr1Mlj
KiJttl7EtTPXLowTnGVCQi+aWEIVmkqhoWTPdhdnzJppDHcGdo0QNtWARbLQJ24r
PzGH4Jw0/wml+9Fsso+ftvGOE+RgYexNWVazCA2xhDuHvx8GybKefC8RrNAKFiWE
aOfGzDgZM/hpTJnGQxSp3systRRV4dmVOdh7/J3rw0X0vGOL5WbhwLIZmriDpGSM
n+Ftc/NQxOf9YaSlxAj3hOL7u1PXt8OMFUVFc4ISz5lJWB7wgU9J+DWW9X5LS/S8
UN/OiPk4L/mwcy8KbfL5dzKfCZiaLY4O+mgv/6skTCReUm/2lvNS+82MuwQK0+u4
0U6znktkAcPJATrCXgQM8Lg3UfWXeISqmP0JKA+jjvx2GMPWQzHhzGer79RXppvj
hYyfMznjrsM1pHNkrR5a1lh+I8segaOU1E3ZlbTXW20spug=
-----END CERTIFICATE-----

Tried to manually import the cert file using mmc.exe - add/remove snappins - certificates. But didn't seem to help.

**Comment by FiloSottile** [comment, marked as answer]
That's a certificate for `museum.test`, not for `xxxx.local`, so you might have configured the wrong certificate.
